packages/core/src/components/Icon/iconHelper.js

```diff
@@ -1,18 +1,21 @@
-import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+// @flow
+import * as React from 'react';
```

Comment:
can do `import React` here

Reply:
in quite a few places in this PR actually. don't know if it's worth changing them all though